config/test/spike-virt-benchmark.toml

```diff
@@ -0,0 +1,23 @@
+# A simple configuration to run on QEMU virt platform
```

Comment:
Small nits, but we should try to update the top-level comment to explain what the config does :)